### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by cevian**
Add proper signup links

### CODE REVIEW

**Review by sgichohi — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -106,7 +106,7 @@ Note: the instructions below are for Timescale's standard compute instance. For
 
 To enable pgvectorscale:
 
-1. Create a new [Timescale Service](https://console.cloud.timescale.com/dashboard/create_services).
+1. Create a new [Timescale account](https://console.cloud.timescale.com/signup?utm_campaign=vectorlaunch).
```
Comment (billy-the-fish) [inline README.md:109]: Can we go with service please. This keeps us aligned with the docs: https://docs.timescale.com/getting-started/latest/services/.

**Review by billy-the-fish — CHANGES_REQUESTED** [review]
Neat link.

**Comment by cevian** [comment]
@billy-the-fish please review again?

**Review by billy-the-fish — APPROVED** [review]
LGTM

**Pull request merged**